code/__DEFINES/preferences.dm

```diff
@@ -36,7 +36,7 @@
 #define BE_ALIEN		64
 #define BE_PAI			128
 #define BE_CULTIST		256
-#define BE_NINJA		512
```

Comment:
Don't think I've forgotten about you, Space Ninja!

- I think I want to look into getting this functional again